.github/workflows/build-and-push.yml

```diff
@@ -23,22 +25,24 @@ jobs:
     runs-on: ubuntu-latest
     env:
       IMAGE_NAME: kserve/modelmesh
-      IMAGE_TAG: latest
     steps:
     - uses: actions/checkout@v2
-    - name: Build runtime image
+    - name: Build and push runtime image
       run: |
         GIT_COMMIT=$(git rev-parse HEAD)
         BUILD_ID=$(date '+%Y%m%d')-$(git rev-parse HEAD | cut -c -5)
 
-        docker build -t ${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }} \
-          --build-arg imageVersion=${IMAGE_TAG} \
+        # Strip git ref prefix from version
+        VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+
+        # Use Docker `latest` tag convention
+        [ "$VERSION" == "main" ] && VERSION=latest
+        echo $VERSION
```

Comment:
We use the same workflow for main/latest and formal releases. Does it make sense to have a separate workflow for releases, as seen in a PR for modelmesh-serving?

Reply:
We will use the same workflow for main/latest and releases, following what kserve is doing.